base/deprecated.jl

```diff
@@ -26,7 +26,7 @@ macro deprecate(old, new, ex=true)
         newname = Expr(:quote, new)
         Expr(:toplevel,
             ex ? Expr(:export, esc(old)) : nothing,
-            :(function $(esc(old))(args...)
+            :(@__doc__ function $(esc(old))(args...)
```

Comment:
Watch out, I snuck in the ability to document deprecated functions :)

Reply:
Oh, never mind, it bails on bootstrapping and I don't feel strongly enough to battle that one out.